### DISCUSSION TITLE
What does the future structure of OMI look like?

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#157] [General]
When we first set out to create the Open Metaverse Interoperability group, we acknowledged that no system is perfect, nor permanent. We believed in collaboration, and iteration, intentionally shaping our culture as we become more diverse. We acknowledged that we would comprise of more than Western contributors, and set out to create a culture that could evolve as we do, while remaining resilient to division before we even got the chance to get started. 

By choosing to see ourselves as a node in a much larger ecosystem, we are by default resilient to many of the temptations, and potential weaknesses of other efforts. We wanted to divide the chairs immediately, to ensure there wouldn't be one person calling the shots. Our group held core values workshops, and elected our first three chairs. We would try out three and iterate from there. To ensure this, we set term length limits of 3 months for the first iteration, 6 months for the second iteration, and will transition into year long representation at our next election. We also intended to adapt our structure as we evolved. With our next election on Feb 2, 2022, we want to look inward at our existing chair structure and explore ways to better serve the Open Metaverse and divide the responsibility of work to represent the objectives of the Open Metaverse Interoperability group. 

This is a larger conversation, that should not occur in isolation. We would like to propose a larger discussion as a group to ensure that the work doesn't fall on a few bottlenecks, and rather, can be divided in a way that unites us all. I have listed out some of the questions and suggestions that have come up over the last 9 months, and hope that we can discuss as a group. 

- We currently have 3 chairs, but are curious - could we restructure to best meet the various needs of our community? 
  - There is development work 
  - There is communications and thought leadership work 
  - There are fiscal and time-allocation-capabilities responsibilities 
  - There are W3C responsibilities
  - There is alignment, strategy and backlogs, operations and collaboration work 
  - There is unique work to different working groups, which is why we operate in working groups 

- Are there benefits to adding _more_ chairs? What could that look like? 
  - Do we want to align our representation around core values as champions for parts of the whole? 
  - Do we want to create more seats to divide and conquer based on the type of work? 
  - How can we align the work so that it is conducive of communal sharing of responsibilities and doesn't fall on the shoulders of a small few? 

- Do we still feel we need 3 W3C chairs? We operate with working groups, which doesn't exactly fit cookie-cutter into the exemplar of a w3c group. We believe W3C responsibilities make up part of our efforts, but may not represent the entirety of our efforts. 
  - Could one or two W3C representatives suffice, with other chairs representing other aspects of the OMI group? 

OMI does not create products or competitors for those creating OMI. Rather we see our work as part of a much larger movement - collaboratively ensuring the Metaverse is open. How can we ensure our core values are carried out and refined without becoming the very thing we set out to prevent in the first place? 

>As a friendly reminder, here are the OMI Core Values. Which should also be revisited periodically to ensure they still reflect our top level objectives. 
>## OMI Core Values: 
>- We collaborate, driven by research, privacy and accessibility, to make the metaverse more human.
>- We work incrementally, iteratively, and with intent.
>- We are creating a cooperative, sustainable culture of innovation.
>- We empower the people making the metaverse.
>- We must cultivate resiliency by championing diversity, and freedom of choice.

I don't expect this to be an easy or simple answer. There is always the possibility that we should keep the existing structure, but that cannot be assumed, and should be agreed upon by the group as a whole. If you have other questions that come to mind, feel free to share them in this discussion as well.

**Comment by mrmetaverse** [comment]
Today term consecutive limits were proposed. I think this is a good idea, but premature for the existing stage of OMI. I think we should consider consecutive term limits of 2 years maybe? With the next election being 1 year, the existing chairs (if re-elected) would have served 1 year and 9 months, and should be eligible for re-election after another term of 1 year. I am wary of anything shorter than 2 years, as we don't want to radically change every year, and I think overlapping service could make for better growth. Thoughts? 

also, I don't think chairs should be the leads of working groups. Or if they are, I don't think chairs should be the leads of more than one working group. We want to avoid creating bottlenecks, and a sign of sustainability is the responsible division of work and accountability.

**Comment by mrmetaverse** [comment]
It was suggested that we draft out a list of the tasks and responsibilities of Chairs and leaders in OMI to help inform a potentially "better" structure.

**Reply by mrmetaverse** [reply]
This issue was created to track the task: https://github.com/omigroup/omigroup/issues/164

**Comment by mrmetaverse** [comment]
Created a new Milestone to divide up this work: [Milestone: Future Structure of OMI
](https://github.com/omigroup/omigroup/milestone/2)

**Comment by robertlong** [comment]
I've created a proposal for new roles in this discussion: https://github.com/omigroup/omigroup/discussions/165

**Comment by mrmetaverse** [comment]
Less chairs and champions, more clearer tasks. What do we need done or want done? Assign people to the tickets. 

In the past year, existing chairs or champions get over/underwhelmed and a lot of these work items tend to get lost. 

People in leadership positions at times get overwhelmed, and those not in leadership positions have felt limited in their capacity to add value. 

A task-based system that is better managed could help. 

We have a task-based system in OMI group, but it never really took flight for individuals of the community to take them on. 

HMW approach this differently? Focus on tasks and not on roles 
- [ ] eliminate all existing tickets and archive them
- [ ]  if a new ticket is created, it needs an assigned owner 
- [ ] if a new ticket is created it needs a deadline (which may change just define that in comments) 
- [ ] accountability: Was this task done? Why so, why not? 
- [ ] maybe bounties as needed

**Reply by mrmetaverse** [reply]
wipe duplicates, but don't wipe away everything. Host a session where others can review the existing tickets, see if anything there needs to be kept and owned. SO that we don't re-write tickets that were there and are ownable.

**Comment by mrmetaverse** [comment]
We may not need additional roles, or to change the roles. Just keep the roles the same, and if folks are burning out, cycle them out of those current roles. Some members are super active but happily focused on one domain (i.e. MSF delegate) and may not want to take over a chair role, and are more happy to stay hyperfocused on their domain. 

Maybe keep the current structure, and focus more on the task board, or whatever we decide is best to track work items.

**Comment by mrmetaverse** [comment]
People enjoy the demo days and world hops. Maybe a structure where once a month we have a demo day or world hop? 

Maybe more structure around that.

**Reply by mrmetaverse** [reply]
Jimmy6dof sees OMI media and events helping with this and has had difficulty getting involved due to inaccurate calendar upkeep.

**Comment by mrmetaverse** [comment]
When we choose people for the next positions, it would be great to have diversity in fields of expertise. 

i.e. we all have our biases (godot, unreal, etc.) and we are better served with diversity of expertises included.
